_data/social.yml

```diff
@@ -93,6 +93,12 @@ websites:
       sms: Yes
       doc: https://help.linkedin.com/app/answers/detail/a_id/531/related/1
 
+    - name: Livejournal
+      url: https://livejournal.com
```

Comment:
The https link appears to redirect to `http://www.livejournal.com/`. I suggest you change the URL to that.